### PR TITLE
DRY-up code having to do with keys and key operations.

### LINKED ISSUE #5: Replace conditionals with map from type => function
Instead of using conditionals to compare a given key type against an array of options, we should have a table of key types that maps directly to the function to call. For example:

```js
  if(_constants.KEY_TYPES.asymmetric.has(type)) {
    return _asymmetricKey.generateKeyPair({keyId, type});
  }
  if(_constants.KEY_TYPES.keyAgreement.has(type)) {
    return _keyAgreementKey.generateKeyPair({keyId, type});
  }
  if(_constants.KEY_TYPES.symmetric.has(type)) {
    return _symmetricKey.generateKey({keyId, type});
  }
```

Should become something like:

```js
// add this helper
function getKeyOp({name, type})) {
  const ops = OPERATIONS[type];
  if(!ops) {
    throw new Error(`Unknown key type "${type}".`);
  }
  const op = ops[name];
  if(!op) {
    throw new Error(`Unsupported operation for key type "${type}".`);
  }
}

// call this code instead
const op = getKeyOp({name: 'generateKeyPair', type});
return op({keyId, type});
```

### PR DESCRIPTION
**Pull request by mattcollier**
Fixes #5.

### CODE REVIEW

**Review by JSAssassin — APPROVED** [review]

File: lib/helpers.js [hunk 1]
```diff
@@ -1,8 +1,38 @@
 /*!
  * Copyright (c) 2021-2022 Digital Bazaar, Inc. All rights reserved.
```
Comment (dlongley) [inline lib/helpers.js:2]:
```suggestion
 * Copyright (c) 2021-2023 Digital Bazaar, Inc. All rights reserved.
```

File: lib/helpers.js [hunk 2]
```diff
@@ -1,8 +1,38 @@
...
+const OPERATIONS = new Map([
+  ['Ed25519VerificationKey2018', _asymmetricKey],
+  ['Ed25519VerificationKey2020', _asymmetricKey],
+  ['urn:webkms:multikey:Ed25519', _asymmetricKey],
+  ['urn:webkms:multikey:P-256', _asymmetricKey],
+  ['urn:webkms:multikey:P-384', _asymmetricKey],
+  ['urn:webkms:multikey:P-521', _asymmetricKey],
+  ['X25519KeyAgreementKey2020', _keyAgreementKey],
+  ['AesKeyWrappingKey2019', _symmetricKey],
+  ['Sha256HmacKey2019', _symmetricKey],
```
Comment (dlongley) [inline lib/helpers.js:20]:
```suggestion
  // asymmetric keys
  ['Ed25519VerificationKey2018', _asymmetricKey],
  ['Ed25519VerificationKey2020', _asymmetricKey],
  ['urn:webkms:multikey:Ed25519', _asymmetricKey],
  ['urn:webkms:multikey:P-256', _asymmetricKey],
  ['urn:webkms:multikey:P-384', _asymmetricKey],
  ['urn:webkms:multikey:P-521', _asymmetricKey],
  // key agreement keys
  ['X25519KeyAgreementKey2020', _keyAgreementKey],
  // symmetric keys
  ['AesKeyWrappingKey2019', _symmetricKey],
  ['Sha256HmacKey2019', _symmetricKey],
```

File: CHANGELOG.md [hunk 3]
```diff
@@ -1,5 +1,10 @@
 # bedrock-ssm-mongodb ChangeLog
 
+## 10.2.0 - 2023-04-xx
+
+### Changed
```
Comment (dlongley) [inline CHANGELOG.md:5]: My read of semver is that this doesn't add any new API surface (nor any new dependency surface) so it's considered a "fix" that simplifies the implementation internally and should therefore be a patch.

```suggestion
## 10.1.3 - 2023-04-xx

### Fixed
```

**Review by dlongley — APPROVED** [review]
Thanks! There are minor suggestions that I'll apply when I merge unless you object to them.

**Comment by mattcollier** [comment]
> Thanks! There are minor suggestions that I'll apply when I merge unless you object to them.

Great, TY.

**Pull request merged**